### PR TITLE
openconnect: update to version 8.02

### PR DESCRIPTION
**Pull request by gpolitis**
###### Type(s)

- [x] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 10.14.3 18D109
Xcode 10.1 10B61

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: net/openconnect/Portfile [hunk 1]
```diff
@@ -31,9 +31,9 @@ depends_lib         port:gettext \
 master_sites        ftp://ftp.infradead.org/pub/openconnect/
 
 # md5 is needed for http master_site below
-checksums           md5     ca2ca1f61b8515879b481dcf6ed4366b \
-                    rmd160  b5f8e916d081011d469f3b180b195113687cdd2b \
-                    sha256  1c44ec1f37a6a025d1ca726b9555649417f1d31a46f747922b84099ace628a03
+checksums           md5     e723c92b0d435df2a521549edbe1fe3e \
+                    rmd160  558b95eae763773aec8c2517027d9680d47ed7e1 \
+                    sha256  1ca8f2c279f12609bf061db78b51e5f913b3bce603a0d4203230a413d8dfe012 \
```
Comment (mf2k) [inline net/openconnect/Portfile]: Please add the `size` parameter.

**Comment by mf2k** [comment]
In the commit message, please add the following to reference the relevant Trac ticket:

`Fixes: https://trac.macports.org/ticket/57995`

**Comment by mf2k** [comment]
Does this version work with the `openconnect-gui` port? Or does that also need updating?

**Comment by gpolitis** [comment]
> Does this version work with the `openconnect-gui` port? Or does that also need updating?

I don't know but I'll check that out (and also address all your comments).

**Comment by mf2k** [comment]
Thank you!

**Pull request merged**